docs/guides/07-zoom-based-styles.md

```diff
@@ -44,23 +43,98 @@ After you explore each one independently, you will bring them together to create
 ### Getting started
 To get started, copy and paste the code below into your favorite text editor and save it to a file `vancouver-trees.html`.
 
+```html
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset="utf-8">
+
+    <script src="../../../dist/carto-vl.js"></script>
+
+    <script src='https://api.tiles.mapbox.com/mapbox-gl-js/v0.50.0/mapbox-gl.js'></script>
+    <link href='https://api.tiles.mapbox.com/mapbox-gl-js/v0.50.0/mapbox-gl.css' rel='stylesheet' />
+
+    <link rel="stylesheet" type="text/css" href="../../style.css">
+    <style>
+        <!-- Zoom level indicator style -->
```

Comment:
I don't think we should focus on the zoom level indicator, that is useful to understand and see how the zoom-based styling works, but I don't expect to see it on final real apps

Reply:
I have removed that comment